selenium_tests/manage_people/page_objects/mp_user_form_page.py

```diff
@@ -19,15 +19,18 @@ def DELETE_USER_ICON(cls, sis_user_id, role):
         locates delete person icon for the given sis_user_id (univ_id) and
         role
         """
-        return By.CSS_SELECTOR, \
-               "li[data-sisid='{}'][data-role='{}'] a.delete-icon".format(
-                   sis_user_id, role)
+        return By.CSS_SELECTOR, "li[data-sisid='{}']" \
+                                "[data-canvas-role-label='{}'] " \
```

Comment:
Can we use `data-role-id` instead, and update the spreadsheet test data? That way we're a little more insulated against name changes in the future.